bedrock/products/templates/products/landing.html

```diff
@@ -15,136 +16,214 @@
   {%- elif LANG == 'en-GB' -%}
     Mozilla’s products — Mozilla (UK)
   {%- else -%}
-    {{ ftl('firefox-products-firefox-beginning', fallback='firefox-products-firefox-is-more-than-a-browser') }} - Mozilla
+    {{ ftl('firefox-products-mozilla-products') }} - Mozilla
   {%- endif -%}
 {%- endblock -%}
 
-{% block page_desc %}{{ ftl('firefox-products-mozillas-family-of-products', fallback='firefox-products-its-a-whole-family-of-products') }}{% endblock %}
+{% block page_desc %}{{ ftl('firefox-products-mozillas-family-of-products') }}{% endblock %}
 
 {% block page_css %}
-  {{ css_bundle('protocol-split')}}
-  {{ css_bundle('protocol-emphasis-box')}}
+  {{ css_bundle('protocol-picto') }}
   {{ css_bundle('products') }}
 {% endblock %}
 
-{% set android_url = play_store_url('firefox', 'products') %}
-{% set ios_url = app_store_url('firefox', 'products') %}
+{% set fx_android_url = play_store_url('firefox', 'products') %}
+{% set fx_ios_url = app_store_url('firefox', 'products') %}
+{% set fc_android_url = play_store_url('focus', 'products') %}
+{% set fc_ios_url = app_store_url('focus', 'firefox-browsers-mobile-focus') %}
+{% set pocket_android_url = play_store_url('pocket', 'products') %}
+{% set pocket_ios_url = app_store_url('pocket', 'products') %}
+
 {% set referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=products' %}
 {% set _entrypoint = 'mozilla.org-products' %}
 
+{% set icon_external = '<span class="mzp-c-button-icon-end"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><g fill="none" fill-rule="evenodd" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"><path d="M13 4h7v7M12 12l8-8M8 4H7a3 3 0 0 0-3 3v10a3 3 0 0 0 3 3h10a3 3 0 0 0 3-3v-1"/></g></svg></span>' %}
```

Comment:
EDIT: I remembered we can just edit `stroke-width` in the svg element ourselves! 

I set the `stroke-width` to be 2.8, and I already see a great improvement on legibility:
![Screenshot 2024-04-30 at 2 04 04 PM](https://github.com/mozilla/bedrock/assets/42309026/83ac10c9-dbad-4514-b825-ef692429eea6)

^ so that would be my new suggestion.